.github/dependabot.yml

```diff
@@ -3,4 +3,38 @@ updates:
 - package-ecosystem: pip
   directory: results-processor/
   schedule:
-    interval: monthly
+    interval: weekly
+- package-ecosystem: gomod
```

Comment:
nit: Add optional reviews, e.g.
    reviewers:
      - "kyleju"

Reply:
let's come back to this one. Maybe we should talk about this being on the task list for engineers on rotation.